en/05_Contributing/17_Managing_Security_Issues.md

```diff
@@ -0,0 +1,113 @@
+
+In those situations, the security team will assess how likely it was that the vulnerability was widely noticed based on how long the issue was opened on GitHub and how explicit the description was. If the risk of the vulnerability having been noticed is small, an owner on the Silverstripe GitHub organisation will delete the public facing issue and create a private one instead.
+
+### Handling issues on Huntr
+
+[Huntr.dev](https://huntr.dev/) is an open source vulnerability bounty platform. The security team occasionally receives vulnerability reports from Huntr. Reports from Huntr go through more or less the process as the regular reports. The security team should take the following points into consideration when working on a Huntr issue:
+
+- While you can reply via email and the response will be recorded in Huntr, who sent the response will not. To keep the discussion flowing more naturally, it is preferable to login into Huntr with your GitHub account and respond via the Huntr interface.
+- People reporting vulnerabilities via Huntr are not always very familiar with Silverstripe CMS. Don't assume that they have a lot of context on what Silverstripe CMS is used for.
+- Some Huntr reporters are just looking to make a quick buck. Review the reporter's profile to see how serious and reliable they are. Occasionally, some reporters will try to open multiple reports for the same underlying vulnerability or push for a higher CVSS score to get a bigger bounty. Do not hesitate to push back if the reporter is being unreasonable.
+- Once we "confirm" an issue in Huntr, it means that we agree the bug exist and that it's a vulnerability. Security enhancement should not be confirmed.
+- By default, Huntr will issue CVEs for issues reported through their platform. We've disabled this functionality.
+
+### A vulnerability has been reported against a dependency of Silverstripe CMS
+
+Silverstripe CMS relies on many other dependencies. Those dependencies will occasionally have vulnerabilities reported and patched against them.
+
+When a vulnerability is disclosed against a Silverstripe CMS dependencies, we generally do not consider this a vulnerability in Silverstripe CMS and will not issue a CVE for it.
+
+We will consider taking actions if:
+- the vulnerability is compounded by the specific way the dependency is used within Silverstripe CMS or
+- a Silverstripe CMS constraint prevents the installation of a patched version of the dependency.
+
+As a general principle, Silverstripe CMS composer constraints do not preclude the installation of dependencies with known vulnerability. It is the responsibility of Silverstripe CMS project owners to regularly update their dependencies and to review the risk posed by specific vulnerabilities.
+
+If you want to preclude the installation of any dependency with known vulnerabilities, you can add the [`roave/security-advisories `](https://packagist.org/packages/roave/security-advisories) packages to your project.
```

Comment:
```suggestion
If you want to preclude the installation of any dependency with known vulnerabilities, you can add the [`roave/security-advisories`](https://packagist.org/packages/roave/security-advisories) package as a dependency to your project.
```
Because this is in a document about how Silverstripe handles security vulnerabilities, it reads as a recommendation that maintainers of Silverstripe CMS can add this as a dependency to supported modules. Perhaps we should just remove this line.

Reply:
Good point ... I've removed the line.